srv/modules/runners/cephprocesses.py

```diff
@@ -17,7 +17,7 @@
 import salt.client
 import salt.utils
 import salt.utils.master
-import six
+import salt.ext.six as six
```

Comment:
I wasn't sure about this change. We import six in other places just fine...ping @jschmid1?